### PR TITLE
Add more spell descs for Pirate

### PR DESCRIPTION
**Pull request by Enochen**
Bottle Toss, Death Glare, Drunken Frenzy, Grog Dance, Pirate Shanty

### CODE REVIEW

File: src/plugins/combat/spells/PirateShanty.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { STRBoost } from '../effects/STRBoost';
 import { DrunkenStupor } from '../effects/DrunkenStupor';
 
 export class PirateShanty extends Spell {
+  static description = 'Sings an inspirational sea shanty with an ally, increasing STR and drunkenness. STR boost scales on # of Bottles. The \"DrunkenStupor\" effect is gained when drunknness reaches 100%.';
```
Comment (seiyria) [inline src/plugins/combat/spells/PirateShanty.js]: don't need to escape double quotes here.

**Comment by seiyria** [comment]
just one small issue, otherwise this looks great!

File: src/plugins/combat/spells/PirateShanty.js (hunk 1 above)
Comment (Enochen) [inline src/plugins/combat/spells/PirateShanty.js]: whoops, I put it there because I was going to use single quotes but then changed my mind

**Comment by seiyria** [comment]
thanks! I'll merge this a bit later.

**Pull request merged**